www/src/components/features/legend-table.js

```diff
@@ -3,95 +3,99 @@ import { jsx } from "theme-ui"
 
 import EvaluationCell from "./evaluation-cell"
 
-const LegendTable = () => {
-  const legendBallStyle = {
-    float: `none`,
-    marginLeft: 0,
-    marginRight: 0,
-    display: `inline-block`,
-  }
+const legendBallStyle = {
+  float: `none`,
+  ml: 0,
+  mr: 0,
+  display: `inline-block`,
+}
 
-  const legendBallCellStyle = t => {
-    return {
-      display: `table-cell`,
-      verticalAlign: `middle`,
-      textAlign: `center`,
-      padding: 3,
-      borderLeft: `1px solid ${t.colors.ui.border}`,
-      borderBottom: `1px solid ${t.colors.ui.border}`,
-    }
+const baseCellStyle = {
+  display: `table-cell`,
+  verticalAlign: `middle`,
+  textAlign: `center`,
+  p: 3,
+}
+
+const legendTableContainerStyle = {
+  border: t => `1px solid ${t.colors.ui.border}`,
+  borderLeft: 0,
+  fontFamily: `heading`,
+}
+
+const legendBallCellStyle = t => {
+  return {
+    ...baseCellStyle,
+    borderLeft: `1px solid ${t.colors.ui.border}`,
+    borderBottom: `1px solid ${t.colors.ui.border}`,
   }
+}
 
-  const legendExplanationCellStyle = t => {
-    return {
-      display: `table-cell`,
-      verticalAlign: `middle`,
-      textAlign: `center`,
-      padding: 3,
-      borderLeft: `1px solid ${t.colors.ui.border}`,
-      borderBottom: [`1px solid ${t.colors.ui.border}`, null, 0],
-    }
+const legendExplanationCellStyle = t => {
+  return {
+    ...baseCellStyle,
+    borderLeft: `1px solid ${t.colors.ui.border}`,
+    borderBottom: [`1px solid ${t.colors.ui.border}`, null, 0],
   }
+}
 
-  const balls = [
-    <div sx={legendBallCellStyle} key={`${legendBallCellStyle}-1`}>
-      <h4 style={{ margin: 0 }}>Icon</h4>
-    </div>,
-    <div sx={legendBallCellStyle} key={`${legendBallCellStyle}-2`}>
-      <EvaluationCell num="3" style={legendBallStyle} />
-    </div>,
-    <div sx={legendBallCellStyle} key={`${legendBallCellStyle}-3`}>
-      <EvaluationCell num="2" style={legendBallStyle} />
-    </div>,
-    <div sx={legendBallCellStyle} key={`${legendBallCellStyle}-4`}>
-      <EvaluationCell num="1" style={legendBallStyle} />
-    </div>,
-    <div sx={legendBallCellStyle} key={`${legendBallCellStyle}-5`}>
-      <EvaluationCell num="0" style={legendBallStyle} />
-    </div>,
-  ]
+const balls = [
```

Comment:
AHHHH WHYYY

okay, there's a lot that can be done here but since this seems to be just for the styles, I'll save it for a future PR.

Reply:
Yes, I only wanted to touch CSS here

Reply:
😆